plasmidfinder/1.0/Dockerfile

```diff
@@ -0,0 +1,70 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the folowing stage. We use `app` for the production image
+FROM debian:stretch as app
+
+#https://bitbucket.org/genomicepidemiology/plasmidfinder/issues/10/version-number
+# ARG sets environment variables during the build stage
+ARG PLASMIDFINDER_VER="2.1"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+# Optional, but highly recommended
+LABEL base.image="debian:stretch"
+LABEL dockerfile.version="1"
+LABEL software="plasmidfinder"
+LABEL software.version=$PLASMIDFINDER_VER
+LABEL description="Identifies plasmids in total or partial sequenced isolates of bacteria."
+LABEL website="https://github.com/StaPH-B/docker-builds"
+LABEL license="https://github.com/StaPH-B/docker-builds/blob/master/LICENSE"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+
+
+ENV DEBIAN_FRONTEND noninteractive,Teletype
```

Comment:
It's better to use
```
ARG DEBIAN_FRONTEND="noninteractive,Teletype"
```
so that it disappears after the build

Reply:
Also, what is the 'Teletype' for?

Reply:
I was using the dockerfile on the tools website as a template. It dose not seem to do anything and I couldn't find anything about it with a google search. So I took it out.